client/src/pages/learn/responsive-web-design/basic-css/index.md

```diff
@@ -26,7 +26,7 @@ The idea behind CSS is that you can use a selector to target an HTML element in
 
 In this section, you'll see how adding CSS styles to the elements of your CatPhotoApp can change it from simple text to something more.
 
-<div role="alert" class="fade alert alert-info show flash-message-enter-active">
+<div role="alert" class="alert alert-info show">
```

Comment:
```suggestion
<div role="alert" class="alert alert-info">
```
Pretty sure the `show` class isn't used for anything here. Considering it is a utility class with `!important` it really should not be applied unless needed.